src/components/Reports/MPGAIncomeExpensesReport/Helper/transformEnums.ts

```diff
@@ -0,0 +1,61 @@
+import { TFunction } from 'react-i18next';
+import {
+  StaffExpenseCategoryEnum,
+  StaffExpensesSubCategoryEnum,
+} from 'src/graphql/types.generated';
+
+export const getLocalizedCategory = (
+  category: StaffExpenseCategoryEnum,
+  t: TFunction,
+) => {
+  return t(`${categoryLabels[category]}`);
```

Comment:
Sorry, but this won't work. The string passed to `t` has to be a static string like `"Donation"` not a variable or a template literal. That's so that `yarn extract` can pull out all of the labels used by the app for translation.

Thanks for renaming the methods though!